spec/integration/cli_verify_spec.rb

```diff
@@ -115,5 +115,26 @@
       expect(result).to exit_with_success(expected_output)
     end
 
+    it "verifies all manifests with invalid topics error" do
```

Comment:
I don't think it's necessary to have an integration test for this validation. It's just another error message, and the code to show error messages already is tested by the integration tests.